components/Fields/Stars.stories.ts

```diff
@@ -0,0 +1,32 @@
+import Stars from './Stars.vue';
+import { bind } from '~/lib/storybook-types'
+
+export default {
+  /* 👇 The title prop is optional.
+  * See https://storybook.js.org/docs/vue/configure/overview#configure-story-loading
+  * to learn how to generate automatic titles
+  */
+  title: 'Stars',
+  component: Stars,
+};
+
+export const Primary = () => ({
+  components: { Stars },
+  template: '<Stars stars="3" />',
+});
```

Comment:
Not required in other.
Really need here?

Reply:
removed